### PR TITLE
Setup.py updates

### PR DESCRIPTION
**Pull request by streeter**

### CODE REVIEW

**Comment by streeter** [comment]
Ready for reviewing.

File: setup.py [hunk 1]
```diff
@@ -1,13 +1,24 @@
 #!/usr/bin/env python
 
+import os
+import sys
+
 from setuptools import setup
 
 
+if sys.argv[-1] == 'publish':
+    os.system('python setup.py register sdist upload')
```
Comment (meastman) [inline setup.py]: Can you combine these into 'register sdist bdist_wheel upload'?

Comment (streeter) [inline setup.py]: Not sure, for some reason I was thinking they needed to be separate.

Comment (meastman) [inline setup.py]: https://pythonhosted.org/setuptools/setuptools.html#upload-upload-source-and-or-egg-distributions-to-pypi says it'll work.

**Comment by meastman** [comment]
:+1:

**Pull request merged**